packages/components/src/internal/schemas.ts

```diff
@@ -112,6 +117,7 @@ export const SCHEMAS = {
     STUDY_TABLES,
     INVENTORY,
     LIST_METADATA_TABLES,
+    PICKLIST_TABLES,
```

Comment:
I know that the feature here is for picklists, but this seem general. Should it just be called `LIST_TABLES`?